Userland/Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -321,6 +322,7 @@ JS::NonnullGCPtr<Document> Document::create(JS::Realm& realm, AK::URL const& url
 
 Document::Document(JS::Realm& realm, const AK::URL& url)
     : ParentNode(realm, *this, NodeType::DOCUMENT_NODE)
+    , m_page(static_cast<Web::Bindings::HostDefined&>(*realm.host_defined()).page)
```

Comment:
we have helpers for the other members of host_defined() like `Bindings::host_defined_environment_settings_object` and `Bindings::host_defined_intrinsics`. can we add one for `host_defined_page` ?